src/style/colors.js

```diff
@@ -99,13 +99,15 @@ class Colors {
 
   getTintedColorForDynamicHex(color, tintKey) {
     // Handles dynamic colors (non uilib colors)
-    let tintLevel = Math.floor(Number(tintKey) / 10 + 1);
-    tintLevel = Math.max(2, tintLevel);
-    tintLevel = Math.min(9, tintLevel);
-    return generateColorTint(color, tintLevel * 10);
+    let tintLevel = Math.floor(Number(tintKey) / 10);
+    tintLevel = Math.max(1, tintLevel);
+    tintLevel = Math.min(8, tintLevel);
+
+    const colorsPalette = this.generateColorPalette(color);
+    return colorsPalette[tintLevel - 1];
```

Comment:
This returns the same color, not the tinted color.

Reply:
are you sure? 
there's a test that shows it works (: 
https://github.com/wix/react-native-ui-lib/blob/master/src/style/__tests__/colors.spec.js#L81

Reply:
I've called it and printed it. I keep getting the same color I sent

Reply:
Ok. My bad. Seems to work